env-check.sh

```diff
@@ -92,9 +106,9 @@ fi
 # octave packages are required only for check-volume-levels.sh
 # Good to check upfront but this can be optional requirement
 out_str="" check_res=0
-func_check_pkg octave
-func_check_pkg octave-signal
-func_check_pkg octave-io
+func_check_dpkg octave
+func_check_dpkg octave-signal
+func_check_dpkg octave-io
```

Comment:
can we add something for Fedora? A number of us use it since it 'just works' on newer devices, and it's also in production for Lenovo devices.

Reply:
We can make this script smarter but is it really worth our time? Right now on Fedora it says "cannot check for octave-io", that's a clear enough message. If we start supporting Fedora here then we need to validate the rest of the script on Fedora too.

Reply:
Whatever happens with Fedora, this PR fixes the check for Ubuntu. It's broken everywhere right now.